xesar-connect/src/main/kotlin/com/open200/xesar/connect/messages/command/SetPhoneNumberOnSmartphoneMapi.kt

```diff
@@ -0,0 +1,14 @@
+package com.open200.xesar.connect.messages.command
+
+import com.open200.xesar.connect.utils.UUIDSerializer
+import java.util.*
+import kotlinx.serialization.Serializable
+
+/** Represents a command POJO to set the phone number on a smartphone media. */
+@Serializable
```

Comment:
can you add the params to the java docs?

Reply:
done :)